build/pr-pipeline.yml

```
@@ -167,7 +167,7 @@ stages:
       resourceGroup: $(ResourceGroupName)
       sqlServerName: $(DeploymentEnvironmentName)
       schemaAutomaticUpdatesEnabled: 'auto'
-      sqlServerAdminPassword: $[stageDependencies.UpdateRandom.Password.outputs['SetVariablesFromRandomString.password']]
+      userAssignedManagedIdentityName: $(DeploymentEnvironmentName)
```

We would need the same logic here so the name matches

```suggestion
      userAssignedManagedIdentityName: "$(DeploymentEnvironmentName)-uami"
```